Backend/ToDoApp/tests/test_views.py

```diff
@@ -8,7 +8,17 @@ def setup(self):
     self.client = Client()
 
   def test_list_tasks_returns_object_with_array_of_tasks(self):
-    task_list_response = self.client.get(reverse('list_tasks'))
+    task_list_response = self.client.get(reverse('task_list'))
     task_list = json.loads(task_list_response.content)
     self.assertEqual(task_list, { 'tasks': [] })
+      
+  def test_new_task_returns_false_with_improper_input(self):
+      response=self.client.post(reverse('add_task'))
+      body=json.loads(response.content)
+      self.assertFalse(body['taskCreated'])
+      
+  def test_new_task_returns_true_with_proper_input(self):
```

Comment:
Same as my comment about improper input.

Reply:
Done. I need to work on being more specific. Thank you for the feedback.